modules/health_quest/app/services/health_quest/appointment_service.rb

```diff
@@ -71,5 +73,9 @@ def other_params(use_cache = false)
     def date_format(date)
       date.strftime('%Y-%m-%dT%TZ')
     end
+
+    def appt_file
```

Comment:
I think ideally we should move away from abbreviations for single words. Thoughts?

Reply:
Agreed! I will change it to appointment_file.

Reply:
Done!